core/backend/src/internal/workspace/WorkspaceImpl.ts

```diff
@@ -339,12 +339,12 @@ class WorkspaceImpl implements Workspace {
   public readonly settings: Settings;
   protected _cloudCache?: WorkspaceCloudCache;
   public getCloudCache(): WorkspaceCloudCache {
-    return this._cloudCache ??= makeWorkspaceCloudCache({ cacheName: "Workspace", cacheSize: "20G" });
+    return this._cloudCache ??= makeWorkspaceCloudCache({ cacheName: workspaceEditorName, cacheSize: "20G" });
```

Comment:
No, the cloud cache for workspaces should be "Workspace". Only the editor should use the workspaceEditorName.

Reply:
reverted. i was confused why i had two CloudCaches. One for Workspace and one for WorkspaceEditor.

Reply:
WorkspaceDbs are always opened readonly in the Workspace cloud cache and writeable when editing. The same WorkspaceDb may be in both.